### PR TITLE
Add replayable.io to readme

### PR DESCRIPTION
**Pull request by ianjennings**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,6 +117,7 @@
 - [Obsidian.md](https://obsidian.md/) - A knowledge base tool that works on local Markdown files. It allows you to create links between different notes.
 - [Scribe](https://scribehow.com/) - Automatically create step-by-step guides for any process. Simply hit “record” and Scribe will generate a detailed guide complete with screenshots based on your actions, ready to share with colleagues, customers, and friends.
 - [Logseq](https://logseq.com/) - Logseq is a privacy-first, open-source knowledge base that works on top of local plain-text Markdown and Org-mode files. Use it to write, organize and share your thoughts, keep your to-do list, and build your own digital garden. 
+- [Replayable](https://replayable.io) - Dashcam for desktop. Remember everything that happens on your desktop.
```
Comment (jyguyomarch) [inline README.md:120]: @ianjennings that's an interesting tool. I think it fits in the list but maybe not the Knowledge Management section. Maybe in the Misc, or maybe we need to start a new Screen Capture section. What do you think?

**Review by jyguyomarch — CHANGES_REQUESTED** [review]
Different or new section needed.

**Review by ianjennings — COMMENTED** [review]
Turn Misc into "screen capture" section

File: README.md (hunk 1 above)
Comment (ianjennings) [reply README.md]: I turned the Misc section into "screen capture" since ShareX is also screen capture.

Comment (jyguyomarch) [reply README.md]: Good idea thx

**Pull request merged**